components/automate-es-gateway/habitat/config/nginx.conf

```diff
@@ -1,7 +1,7 @@
 daemon off;
 pid {{ pkg.svc_var_path }}/pid;
 worker_processes {{ cfg.ngx.main.worker_processes }};
-error_log {{ cfg.ngx.main.error_log }} {{ cfg.logger.level }};
+error_log {{ cfg.ngx.main.error_log }} {{ cfg.log.level }};
```

Comment:
This is not related to the PR but found a bug with the config related to nginx log path